specification/automation/resource-manager/Microsoft.Automation/preview/2022-06-30-preview/runtime.json

```diff
@@ -0,0 +1,399 @@
+            "description": "Automation error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "Runtime"
+        ],
+        "operationId": "Runtime_Get",
+        "description": "Retrieve a runtime.",
+        "externalDocs": {
+          "url": "http://aka.ms/azureautomationsdk/runtimeconfigurationoperations"
+        },
+        "x-ms-examples": {
+          "Get a runtime": {
+            "$ref": "./examples/getRuntime.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../common/v1/definitions.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../common/v1/definitions.json#/parameters/AutomationAccountNameParameter"
+          },
+          {
+            "name": "runtimeName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The runtime name"
+          },
+          {
+            "$ref": "../../common/v1/definitions.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../common/v1/definitions.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/Runtime"
+            }
+          },
+          "default": {
+            "description": "Automation error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "Runtime"
+        ],
+        "operationId": "Runtime_Create",
+        "description": "Create a runtime.",
+        "externalDocs": {
+          "url": "http://aka.ms/azureautomationsdk/runtimeconfigurationoperations"
+        },
+        "x-ms-examples": {
+          "Put a runtime": {
+            "$ref": "./examples/createRuntime.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../common/v1/definitions.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../common/v1/definitions.json#/parameters/AutomationAccountNameParameter"
+          },
+          {
+            "name": "runtimeName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The runtime name"
+          },
+          {
+            "name": "RuntimeCreationParameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/RuntimeCreateOrUpdateParameters"
```

Comment:
>RuntimeCreateOrUpdateParameters [](http://example.com/codeflow?start=37&length=31)

Same comment as your dependency PUT API

Reply:
For runtime resource, I have used the same model for PUT and GET.